mysql-test/suite/rocksdb/t/deadlock_stats.test

```diff
@@ -0,0 +1,43 @@
+CREATE TABLE t1 (
+                   id int primary key, 
+                     value int
+                ) engine=rocksdb;
```

Comment:
for this stat it seems that we have added support for collecting the stats in both `innodb` and `rocksdb`.  But I don't see the tests that verify the `innodb` stats collection works properly.